### PR TITLE
ISPN-13091 Upgrade to 12.1.4.Final

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-13091

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,8 +8,8 @@
     <name>Infinispan Tutorial: Simple tutorials</name>
 
     <properties>
-       <version.infinispan>12.1.3.Final</version.infinispan>
-       <version.protostream>4.4.1.Final</version.protostream>
+       <version.infinispan>12.1.2.Final</version.infinispan>
+       <version.protostream>4.4.0.Final</version.protostream>
```
Comment (pruivo) [inline pom.xml:12]:
```suggestion
       <version.infinispan>12.1.4.Final</version.infinispan>
       <version.protostream>4.4.1.Final</version.protostream>
```

Comment (karesti) [reply pom.xml:12]: I don't know what happened here xD

**Comment by karesti** [comment]
@pruivo rebased

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**